admin-console/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>admin-console</artifactId>
```

Comment:
The `artifactId` of other modules is `notifications-*`. Should this one be named `notifications-admin-console`? (without changing the sources folder name that would still be `admin-console`)

Reply:
Sounds good